rules/tines_rules/tines_global_resource_destruction.yml

```diff
@@ -0,0 +1,26 @@
+AnalysisType: rule
+DisplayName: "Tines Global Resource Destruction"
+Enabled: true
+Filename: tines_global_resource_destruction.py
+Severity: Low
+Tests:
+    - ExpectedResult: true
```

Comment:
This needs a negative test case to demonstrate that it doesn't fire on every event fed to it.